datafusion/src/execution/context.rs

```diff
@@ -2907,6 +2947,29 @@ mod tests {
         assert_eq!(Weak::strong_count(&catalog_weak), 0);
     }
 
+    #[tokio::test]
+    async fn sql_create_schema() -> Result<()> {
+        // the information schema used to introduce cyclic Arcs
```

Comment:
not sure what this comment means

Reply:
oops that may have been straggling comment from a copy paste, sry about that.  i can remove in subsequent PR